cmd/limactl/backup.go

```diff
@@ -0,0 +1,52 @@
+package main
+
+import (
+	"fmt"
+
+	"github.com/lima-vm/lima/pkg/snapshot"
+	"github.com/lima-vm/lima/pkg/store"
+
+	"github.com/spf13/cobra"
+)
+
+func newBackupCommand() *cobra.Command {
+	var backupCmd = &cobra.Command{
+		Use:               "backup INSTANCE",
+		Short:             "Backup (save) an instance",
```

Comment:
I'd expect "backup" to back up the VM to an external storage.

Maybe it should be like `limactl snapshot (ls|create|rollback)` ?

Reply:
Might want to use the same words as qemu, in that case. (list/create/apply/delete)

I can make a go at implementing the "list", changing the cmd should be easier to do

Reply:
Possibly should also try to implement "off-line" versions, of the HMP commands ?

```
FATA[0000] expected status "Running", got "Stopped"     
```

       Parameters to snapshot subcommand:

       snapshot
           is the name of the snapshot to create, apply or delete

       -a  applies a snapshot (revert disk to saved state)

       -c  creates a snapshot

       -d  deletes a snapshot

       -l  lists all snapshots in the given image

So that one can perform those operations with `qemu-img`, when the VM is not running ?

Reply:
The list doesn't have any fancy output options, so probably just dump the output as-is ?

```
Snapshot list:
ID        TAG                     VM SIZE                DATE       VM CLOCK
1         tag                         0 B 2022-09-21 18:31:41   00:00:00.000
```